### PR TITLE
gxm: Name some unknown fields

### PR DESCRIPTION
**Pull request by bythos14**

### CODE REVIEW

File: vita3k/gxm/include/gxm/types.h [hunk 1]
```diff
@@ -1336,14 +1336,24 @@ using SceGxmVertexOutputTexCoordInfos = std::array<uint8_t, 10>;
...
 struct SceGxmProgramVertexVaryings {
-    std::uint8_t unk0[8];
-    std::uint8_t fragment_output_start; // this might be wrong
-    std::uint8_t unk1;
-    std::uint8_t output_param_type;
-    std::uint8_t output_comp_count;
-
-    std::uint16_t varyings_count;
-    std::uint16_t pad0; // padding maybe
+    union {
```
Comment (sunho) [inline vita3k/gxm/include/gxm/types.h:1339]: Is this really union? Didn't you intend to replace unk0 with attrib_pa_regs and untyped_pa_regs?

File: vita3k/gxm/include/gxm/types.h [hunk 2]
```diff
@@ -1404,7 +1414,7 @@ struct SceGxmProgram {
 
     std::uint8_t major_version; //min 1
     std::uint8_t minor_version; //min 4
-    std::uint16_t unk6; //maybe padding
+    std::uint16_t sdk_version; // 0x350 - 3.50
```
Comment (sunho) [inline vita3k/gxm/include/gxm/types.h:1417]: oh cool

File: vita3k/gxm/include/gxm/types.h (hunk 1 above)
Comment (sunho) [reply vita3k/gxm/include/gxm/types.h:1339]: Something you might want to consider is SceGxmProgramVertexVaryings of fragment program is the input vertex attributes. Maybe it's just better to change the name of this struct. Although I'm not sure about unk0 of vertex program.

Comment (sunho) [reply vita3k/gxm/include/gxm/types.h:1339]: Oh I just noticed that you only marked some part of struct as union. varyings_count should be outside of union. It's defined in fragment program.

Comment (sunho) [reply vita3k/gxm/include/gxm/types.h:1339]: Alright top one is for vertex and bottom one is for fragments I see. Ignore all my comments lol.

Comment (bythos14) [reply vita3k/gxm/include/gxm/types.h:1339]: 👍

Comment (bythos14) [reply vita3k/gxm/include/gxm/types.h:1339]: Ok, I did that and also named unk18.

**Review by Zangetsu38 — DISMISSED** [review]

**Comment by bythos14** [comment]
Named one more field, unk18 in SceGxmProgram.

**Comment by bythos14** [comment]
Oops, sorry. That should be all I've got for now.

**Pull request merged**